content/03.results.md

```diff
@@ -222,7 +222,7 @@ While we did not find a _known_ pathogenic variant in the germline of PT_VTM2STE
 This is not surprising since an estimated 49% of germline _PMS2_ variants in patients with CMMRD and/or Lynch syndrome are VUS [@doi:10.1136/jmedgenet-2020-107627].
 Interestingly, while the cell line derived from patient PT_VTM2STE3's tumor at progression was not hypermutated (TMB = 5.7 Mut/Mb), it solely showed the MMR2 signature of the eight CNS signatures examined, suggesting selective pressure to maintain a mismatch repair (MMR) phenotype _in vitro_.
 From patient PT_JNEV57VK, only one of the two cell lines derived from the progressive tumor was hypermutated (TMB = 35.9 Mut/Mb).
-This hypermutated cell line was strongly weighted towards signature 11, while this patient's non-hypermutated cell line showed a number of lesser signature weights (1, 11, 18, 19, MMR2), highlighting the plasticity of mutational processes and the need to carefully genomically characterize and select models for preclinical studies based on research objectives.
+This hypermutated cell line was strongly weighted towards signature 11, while this patient's non-hypermutated cell line showed a number of lesser signature weights (1, 11, 18, 19, MMR2; Table S2), highlighting the plasticity of mutational processes and the need to carefully genomically characterize and select models for preclinical studies based on research objectives.
```

Comment:
I wanted to make sure we pointed folks to where they could find this data, since it's not in the table itself.